src/galaxy/SystemBody.cpp

```diff
@@ -250,6 +250,100 @@ bool SystemBody::HasAtmosphere() const
 	return (m_volatileGas > fixed(1, 100));
 }
 
+double SystemBody::GetPlanckBrightness(const double wavelength_nm, const int temperature) const
+{
+	// Planck's constant
+	double h = 6.62607015e-34; // kg*m^2*s^-1 (J*s)
```

Comment:
These single letter constants can all be made `static const` or probably better `constexpr`

Reply:
Actually going further than that, since `GetPlanckBrightness` is a const method it could leave the class entirely and sit in the cpp just above the `GetColor` implemention, or in a namespace at the top of the file. Unless you want it accessible elsewhere.